tests/src/test/java/alluxio/shell/AlluxioShellUtilsTest.java

```diff
@@ -65,14 +65,29 @@ public void getFilePath() throws IOException {
     }
   }
 
+  /** Type of file system. */
   public enum FsType {
     TFS, LOCAL
   }
 
+  /**
+   * Reset the file hierarchy.
+   *
+   * @return the test directory
+   * @throws IOException
```

Comment:
we generally don't javadoc exceptions for test classes because the exceptions are never supposed to happen.